editor/selectors.js

```diff
@@ -226,9 +242,26 @@ export function isEditedPostBeingScheduled( state ) {
  * @return {String}       Raw post title
  */
 export function getEditedPostTitle( state ) {
-	return state.editor.edits.title === undefined
-		? get( state.currentPost, 'title.raw' )
-		: state.editor.edits.title;
+	const editedTitle = get( state.editor, 'edits.title' );
```

Comment:
A few minor thoughts:

- `get` with string as the second argument is significantly slower than array alternative ([benchmark](https://jsperf.com/lodash-get-string-vs-array))
- `state.editor.edits` is always an object (based on reducer default value) and we don't need `get` to test its truthiness, for a trivial performance improvement: `state.editor.edits.title`
- We should compose selectors vs. accessing state directly when possible, i.e. `getPostEdits` instead of `state.editor.edits`, `getCurrentPost` instead of `state.currentPost`
- `get` accepts a third argument for default value, so this could be expressed as a single `return` value, though maybe not as readable